src/components/InviteForm.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { shareList } from '../api';
+import { useAuth } from '../api/useAuth.jsx';
+
+const InviteForm = ({ listPath }) => {
+	const [input, setInput] = useState({
+		recipientEmail: '',
+	});
+	const { user } = useAuth();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			await shareList(listPath, user.uid, input.recipientEmail);
+			setInput({ recipientEmail: '' });
+		} catch (err) {
+			console.error(err);
+		}
+	};
+	const handleInputChange = (e) => {
+		const { name, value } = e.target;
+		setInput((prevState) => ({
+			...prevState,
```

Comment:
If you use a string instead of an object you could just do `setInput(e.target.value)` and save a few lines of code. Wondering if using an object doesn't make the most sense since there is only a single value and this could be a bit verbose for what you're trying to achieve? Just a thought!